### ISSUE TITLE
Running filters for 'crane status'

### ISSUE THREAD

**Issue by saidimu** [michaelsauter/crane#189]
I propose a flag to `crane status` which filters the output by a container's running status.

By default all containers defined in crane.yml are displayed, with the last column showing the runing status:

```
NAME  IMAGE  ID  UPTODATE  IP  PORTS  RUNNING
```
When one has a lot of non-running containers, a common-case with data-only containers, the output is mostly blank noise:

```
searchdata    busybox  -  -  -  -  -
postgresdata  busybox  -  -  -  -  -
queuedata     busybox  -  -  -  -  -
```

Perhaps such a flag already exists? Couldn't find anything in the  `v1.2.0` docs.

Given that the outputs of `crane status -v` and `crane status` are identical, a new flag isn't strictly needed as the current `crane status` output can be seen as the verbose option. The disadvantage of this is that it breaks existing behavior.

**Comment by bjaglin** [comment]
I can confirm that no such flag exists as of v1.2.0. I kind of like the idea, but I am not sure everyone would agree on what could be filtered out by default: non-running containers or non-existing containers?

**Comment by michaelsauter** [comment]
I agree with @bjaglin, the filters would have to be explicit. `-v` is supposed to give more background information, not to alter the actual output/action.

I would be open to filters for the status command, but they have very low priority for me. If you would like to see them in Crane, please contribute them.
